### ISSUE TITLE
Alarm state of Indoor 2K camera is not changed via Home app

### ISSUE THREAD

**Issue by njuart** [bropat/eufy-security-client#161]
### Client version

2.0.1

### Node version

16.15.0

### Operating System type

Linux

### Operating system version

Raspbian 11

### Describe the bug

Hi, hope I’m addressing issue in a right way. In my setup I’m using Indoor 2K camera integrated into Homekit via Homebridge Eufy Security plugin. Recently one weird problem appeared. When I change alarm state of camera to one of the options (away, home etc.) it doesn’t change and gets stuck in infinite arming loop. But when I immediately open Eufy app and check if state is changed there (and it’s actually changed) and then go back to Home app I see that state now changed there also. It happens every time. I don’t have this problem with arming Homebase 2. We’ve made some testing and research here https://github.com/homebridge-eufy-security/plugin/issues/38 and came to a conclusion that issue is not related to plugin itself, but may be somehow related to Eufy Security Client. So maybe there are any suggestions regarding this. Thx in advance.

### To reproduce

1. Arm Indoor 2K camera alarm in Home app (via Homebridge Eufy Security plugin
2. See infinite arming loop
3. Go to Eufy app
4. Check guard state there
5. See it changed
6. Go back to Home app
7. See it’s changed there also

### Screenshots & Logfiles

![4359519C-413F-46DC-94E0-8B44F8399705](https://user-images.githubusercontent.com/21228802/171504627-79fd249f-616e-4965-a3b2-642e80c32367.jpeg)

### Additional context

Additional investigation can be found here https://github.com/homebridge-eufy-security/plugin/issues/38

**Comment by Palmke** [comment]
I get events when switching the mode of my indoor cams without issues.
Which property are you listening to, `guardMode` or `currentMode`? I noticed only getting updates for the latter when using a T8410.

**Comment by thieren** [comment]
Hey @bropat 

Just checked with @njuart and unfortunately this doesn't seem to fix the issue. Any other ideas?

**Comment by bropat** [comment]
Then I need the debug logs from @njuart (eufy-security-client).

**Comment by thieren** [comment]
Sure. Will check with him and we'll provide the logs

**Comment by bropat** [comment]
It looks like the T8400 does not give this feedback over the P2P connection, unlike e.g. T8410 or T8010:

```
2022-06-13T09:34:28.200Z - DEBUG: Station T8010P************ - DATA CONTROL - received from host **.**.***.*:10134 - Processing sequence 0...
2022-06-13T09:34:28.200Z - DEBUG: Station T8010P************ - Received data{"seqNo":0,"header":{"commandId":1151,"bytesToRead":8,"channel":255,"signCode":0,"type":0},"bytesRead":8,"bytesToRead":8,"firstPartMessage":true,"messageSize":24}
```

Or only this part is missing in the log extract above ;)
Would it be possible to get a complete log?

**Comment by Palmke** [comment]
I also make the same observation in my T8140's.
Only when opening the app I see it updates. So the eufy app itself also needs to poll the information from the device. 

It appears the homebase is more talkative because of the alarm system it providers. Hence the recent adoptations of alarm armed/delay events in this repository. It must send out this data for the external monitoring service eufy providers in some markets.

**Comment by thieren** [comment]
> It looks like the T8400 does not give this feedback over the P2P connection, unlike e.g. T8410 or T8010:
> 
> ```
> 2022-06-13T09:34:28.200Z - DEBUG: Station T8010P************ - DATA CONTROL - received from host **.**.***.*:10134 - Processing sequence 0...
> 2022-06-13T09:34:28.200Z - DEBUG: Station T8010P************ - Received data{"seqNo":0,"header":{"commandId":1151,"bytesToRead":8,"channel":255,"signCode":0,"type":0},"bytesRead":8,"bytesToRead":8,"firstPartMessage":true,"messageSize":24}
> ```
> 
> Or only this part is missing in the log extract above ;) Would it be possible to get a complete log?

Actually both are the last log messages of the logs. So no additional output we could provide.
To my knowledge @njuart waited a while after setting the mode until he pulled the log. So yeah. No feedback over p2p connection from the 'station'.

**Comment by bropat** [comment]
I have understood where the problem lies. Thank you for the hints! I will fix it.

**Comment by thieren** [comment]
@njuart reports that your changes indeed fix the issue. Superb work!

**Comment by jhaleit** [comment]
I have a T8414 and this issue still exists. Changing the guard mode doesn’t do anything besides time out..

**Comment by thieren** [comment]
Hi @jhaleit 

The fix will be implemented in version 2.1.1. until now 2.1.0 is the published version of this library.
